boards/nxp/mimxrt1180_evk/mimxrt1180_evk.dtsi

```diff
@@ -12,6 +12,7 @@
 		led0 = &green_led;
 		sw0 = &user_button;
 		pwm-led0 = &green_pwm_led;
+		watchdog0 = &rtwdog0;
```

Comment:
Set this at the SOC level devicetree, since the watchdog is internal to the SOC. Example: https://github.com/zephyrproject-rtos/zephyr/blob/e60f04096cd8d9227c8279ca830e72bc1b5b8b1c/dts/arm/nxp/nxp_mcxw71.dtsi#L17